samples/helloworld_esm/worker.js

```diff
@@ -4,6 +4,82 @@
 
 export default {
   async fetch(req, env) {
-    return new Response("Hello World\n");
+    let resp = await fetch("https://example.com");
```

Comment:
Don't forget to revert this sample back to its original state when this is done. Most likely it would make sense to have this as a separate example.